### DISCUSSION TITLE
Can't parse neither dates nor numbers

### DISCUSSION THREAD

**Discussion by nareto** [grafana/grafana-infinity-datasource#278] [Q&A]
I'm having trouble with this data format:

```json
{
"data" : [ ["1643389200.000000000",null,null,null,"726",null,null,null,null,null,null,null,null,null,null,null,null,null],
["1643392800.000000000",null,null,null,"546",null,null,null,null,null,null,null,null,null,null,null,null,null],
["1643396400.000000000",null,null,null,"664",null,null,null,null,null,null,null,null,null,null,null,null,null],
["1643400000.000000000",null,null,null,"1432",null,null,null,null,null,null,null,null,null,null,null,null,null],
["1643403600.000000000",null,null,null,"1492",null,null,null,null,null,null,null,null,null,null,null,null,null] ]
}
```

The list of lists in `data` represents the data I want to plot as a time series, with the time given by the first column (UNIX timestamp, seconds). As far as I understand I should be able to select the first column with `$.data[*][0]` and similarly for the others, but I keep getting errors. 

Anyone can help?

Here is a screenshot:

![grafik](https://user-images.githubusercontent.com/461518/156807726-6cfd9fab-4ab2-4949-8787-687bcb54bccd.png)

This BTW comes from the Snowflake API which wraps all values in double quotes.

**Comment by yesoreyeram** [comment]
I think you need this.. https://grafana-infinity-datasource.herokuapp.com/explore?orgId=1&left=%7B%22datasource%22:%22Infinity%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22type%22:%22json%22,%22source%22:%22inline%22,%22format%22:%22table%22,%22url%22:%22https:%2F%2Fjsonplaceholder.typicode.com%2Fusers%22,%22url_options%22:%7B%22method%22:%22GET%22,%22data%22:%22%22%7D,%22root_selector%22:%22data%22,%22columns%22:%5B%7B%22text%22:%22Time%22,%22selector%22:%220%22,%22type%22:%22timestamp_epoch_s%22%7D,%7B%22text%22:%22Value%22,%22selector%22:%224%22,%22type%22:%22number%22%7D%5D,%22filters%22:%5B%5D,%22global_query_id%22:%22%22,%22data%22:%22%7B%5Cn%5C%22data%5C%22%20:%20%5B%20%5B%5C%221643389200.000000000%5C%22,null,null,null,%5C%22726%5C%22,null,null,null,null,null,null,null,null,null,null,null,null,null%5D,%5Cn%5B%5C%221643392800.000000000%5C%22,null,null,null,%5C%22546%5C%22,null,null,null,null,null,null,null,null,null,null,null,null,null%5D,%5Cn%5B%5C%221643396400.000000000%5C%22,null,null,null,%5C%22664%5C%22,null,null,null,null,null,null,null,null,null,null,null,null,null%5D,%5Cn%5B%5C%221643400000.000000000%5C%22,null,null,null,%5C%221432%5C%22,null,null,null,null,null,null,null,null,null,null,null,null,null%5D,%5Cn%5B%5C%221643403600.000000000%5C%22,null,null,null,%5C%221492%5C%22,null,null,null,null,null,null,null,null,null,null,null,null,null%5D%20%5D%5Cn%7D%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D

**Reply by nareto** [reply]
screenshot for reference:
![grafik](https://user-images.githubusercontent.com/461518/156881468-e1f614c9-45de-460a-a027-386e1dfbd3af.png)

Thanks @yesoreyeram, that's definitely progress: the table now displays properly. However the time series graph is still empty (even if I switch to "Time Series" format in the infinity query panel). Why is that?

Also, I was under the impression from the docs that I should be using JSONPath, but that seems not to be the case. The syntax you used, where is it documented?
